oauth2client/contrib/appengine.py

```diff
@@ -89,11 +89,11 @@ def _safe_html(s):
     return cgi.escape(s, quote=1).replace("'", '&#39;')
 
 
-class InvalidClientSecretsError(Exception):
-    """The client_secrets.json file is malformed or missing required fields."""
+class Error(Exception):
+    """Base error for this module."""
```

Comment:
I figured that it may be a good idea to have a module base error to inherit from versus the base python `Exception`.  With only one other module exception, it may be considered overkill.  If there were multiple module exceptions, I think it would be best to have a module base exception.